source/documentation/administration_guide/topics/Editing_Vfs_on_NIC.adoc

```diff
@@ -19,13 +19,13 @@ By default, all virtual networks have access to the virtual functions. You can d
 
 .Prerequisite
 
-* For a vNIC to be attached to a VF must, its passthrough property must be enabled. For details, see xref:Enabling_Passthrough_on_a_vNIC_Profile[].
+* For a vNIC to be attached to a VF must, its passthrough property must be enabled. For details, see xref:Enabling_Passthrough_on_a_vNIC_Profile[Enabling_Passthrough_on_a_vNIC_Profile].
 
 .Procedure
 
 . Click menu:Compute[Hosts].
 . Click the name of an SR-IOV-capable host. This opens the details view.
-. Click the *Network Interfaces* tab.
+. Click the *Network Interfaces* tab.``
```

Comment:
This `` looks like an error?

Reply:
Yup. Thanks. I tend to hit the ` key when I'm trying to hit "Esc".

Reply:
I'll do a global search for that.

OMG. I actually found code blocks marked with ```! We're not supposed to use that. It's from markdown, not Asciidoc.